api/src/main/java/javax/mvc/engine/ViewEngineContext.java

```diff
@@ -74,21 +71,6 @@
     Models getModels();
 
     /**
-     * Returns HTTP request object from the Servlet container.
-     *
-     * @return HTTP request object.
-     */
-    HttpServletRequest getRequest();
-
-    /**
-     * Returns HTTP response object from the servlet container. The underlying
-     * output stream should be used to write the result of processing a view.
-     *
-     * @return HTTP response object.
-     */
-    HttpServletResponse getResponse();
-
```

Comment:
You're absolutely right! Well spotted!
I see that some of the ViewEngine extensions for Ozark use `response.getWriter()`. If we expose the   `OutputStream` through the context, do we need the writer as well then or is it enought with the OS?

I'll also look at the headers. Probably a setter, as you suggest, is the better option?